UIViewExtension.swift

```diff
@@ -0,0 +1,53 @@
+
+/*
+* JTExtensions
+* UIView
+*
+* Description:
+* Extensions to make working with UIViews, as well as UIView subclasses
+* more simple, intuative and powerful.
+*
+* Jtribe Copyright © 2016
+*/
+
+import UIKit
+
+// MARK: Positioning & Sizing
+
+// Description:
+// This extension provides quicker and easier access to the position and
+// size values relating to a UIView
+
+extension UIView {
+	
+	var x: CGFloat {
+		get{ return self.frame.origin.x }
+		set(newValue){ self.frame.origin.x = newValue }
+	}
+	
+	var y: CGFloat {
+		get{ return self.frame.origin.y }
+		set(newValue){ self.frame.origin.y = newValue }
+	}
+	
+	var width: CGFloat {
+		get{ return self.frame.size.width}
+		set(newValue){self.frame.size.width = newValue }
+	}
+	
+	var height: CGFloat {
+		get{ return self.frame.size.height }
+		set(newValue){ self.frame.size.height = newValue }
+	}
+	
+	var origin: CGPoint {
```

Comment:
Yeah considered naming it that. There's no origin property on UIView though, so there is no conflict. I chose `origin` to keep it consistent, as that is the name of the property on the `frame` object that is being changed. What does @mark-jTribe think?